### PR TITLE
[vrops-exporter] Updated VcsaSSLCertificateExpiry alerts

### PR DESCRIPTION
**Pull request by christopherhans**

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/vcenter.alerts [hunk 1]
```diff
@@ -2,8 +2,9 @@ groups:
 - name: vcenter.alerts
   rules:
   - alert: VcsaSslCertificateExpiryWarning
-    expr: >
-      vrops_vcenter_vcsa_certificate_remaining_days < 31 and vrops_vcenter_vcsa_certificate_remaining_days > 15
+    expr: |
+      (vrops_vcenter_summary_certificate_expiry_date / 1000) - time() < (31*86400) 
+      and on(vcenter) (vrops_vcenter_summary_certificate_expiry_date / 1000) - time() > (15*86400)
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/vcenter.alerts:7]: Sophisticated. But I would rather stay with days like so:
```suggestion
    expr: 15 > round(((vrops_vcenter_summary_certificate_expiry_date / 1000) - time()) / 86400) < 31
```

File: prometheus-exporters/vrops-exporter/alerts/vcenter.alerts [hunk 2]
```diff
@@ -17,7 +18,7 @@ groups:
       summary: "SSL Certificate of vCSA {{ $labels.vcenter}} expires in {{ $value }} days. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
 
   - alert: VcsaSslCertificateExpiryCritical
-    expr: vrops_vcenter_vcsa_certificate_remaining_days < 16
+    expr: (vrops_vcenter_summary_certificate_expiry_date / 1000) - time() < (16*86400)
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/vcenter.alerts:21]:
```suggestion
    expr: round(((vrops_vcenter_summary_certificate_expiry_date / 1000) - time()) / 86400) < 16
```

**Review by richardtief — COMMENTED** [review]
Added a slight improvement for better readability.

File: prometheus-exporters/vrops-exporter/alerts/vcenter.alerts (hunk 1 above)
Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/vcenter.alerts:7]: I like it!

**Review by richardtief — APPROVED** [review]
Thanks!

**Pull request merged**